packages/perspective-viewer-d3fc/src/js/legend/scrollableLegend.js

```diff
@@ -102,6 +121,13 @@ export default fromLegend => {
         return scrollableLegend;
     };
 
+    scrollableLegend.useDynamicPageSize = (selection, resizedOnAxes) => {
+        if (resizedOnAxes.vertical === true) {
+            calculatePageSize(selection);

```

Comment:
Is there any reason we don't *always* do `calculatePageSize`? i.e. we could do this up on line 30 in the standard render code (it'll just mean it measures the container on first render too).

That would mean we could do away with the `useDynamicPageSize` function completely, then in the `resizable.onResize()` callback you'd just need to call `scrollLegend(legendSelection)` to re-render it.

Reply:
As discussed, was to achieve a nice sensible default container size. Instead we'll do this with css to simplify the code.